### ISSUE TITLE
Tabhunter stopped working

### ISSUE THREAD

**Issue by hd46510** [ericpromislow/tabhunter#107]
Hello, 
Tabhunter stopped working about 3 weeks ago. 
It doesnt list any of the open tabs, saying there are 0 tabs open (hunting through 0 tabs), despite there are several open. 
What has happened? Thanks in advance!

**Comment by ericpromislow** [comment]
Are there any error messages in the console?

Also, I've noticed that firefox (on the mac) lately sometimes has a severe
memory leak. Just before I have to reboot the machine, the tabhunter panel
is completely empty -- no tabs listed, and I believe the button labels are
gone as well. Maybe you're hitting this.

**Comment by hd46510** [comment]
Thank you for your answer (and for developing tabhunter, of course! ;-) )

I am afraid I am too much of a simple user, because I don't know which console is meant.
Could you please guide me in the right direction?

This is only a guess, but it may have to do with the latest firefox update on june, 3rd to version 77.0.1
But this can also be a coincident, as I cannot tell exactly when tabhunter worked for me the last time.

**Comment by ericpromislow** [comment]
Tools | Web Developer | Browser Console

The 'Web Developer' tools might be optional, but I've had them running
forever and don't know how to enable them.

Separate question: do you now have to periodically shut down Firefox or
even reboot your machine due to running out of memory? Because that's when
I saw the tabhunter stop working.

**Comment by hd46510** [comment]
Thanks, now I found the console.
With an emtpy error list, I get these entries, when the Tabhunter icon is clicked to show the Tabhunter window.
(The last 4 entries appear not before the button is clicked again to close the tabhunter window)
![errors firefox browser console](https://user-images.githubusercontent.com/67317869/85898869-58a46880-b7fd-11ea-8bcc-69f286efd6bc.jpg)

Regarding Memory problems: No, there weren't any of these issues.

**Comment by ericpromislow** [comment]
Do you have any other panel-like extensions you can try?  For example,
AdNauseum.

At a glance, it looks like there's a problem with your environment, and
Tabhunter is reporting its inability to do basic operations.

**Comment by hd46510** [comment]
Weird stuff was going on in my firefox. 
Anyway, Tabhunter ist working again!

I supspected the latest firefox update of being the date, where tabhunter stoped working.
Indeed this seems to have been the problem.

I noticed that some other add-ons were not working also and began searching and found this info
https://www.reddit.com/r/firefox/comments/gzu864/many_things_broken_after_firefox_7701_64bit_update/
This seems to have solved it, everything is working as before.

Thank you for your time and best greetings!
